README.md

```diff
@@ -18,6 +18,12 @@ Running on the *host* interface (recommended):<br/>
 $ docker run -d --net=host --name=squad-dedicated cm2network/squad
 ```
 
+Running as a normal docker container:<br/>
```

Comment:
Please rename to "Running on *docker0* (slower):<br/>.

There are plenty of benchmarks showing that the docker bridge can be a bottleneck when it comes to bandwidth, a thing we should warn people about especially ones running multiple game servers on their machines.